app/_src/networking/transparent-proxy/reachable-backends.md

```diff
@@ -0,0 +1,185 @@
+    redirectPortOutbound: {{ tproxy.defaults.redirect.outbound.port }}
+    reachableBackends:
+      refs:
+      - kind: MeshService
+        name: redis
+      - kind: MeshMultiZoneService
+        labels:
+          kuma.io/display-name: test-server
+      - kind: MeshExternalService
+        name: mes-http
+```
+{% endtab %}
+{% endtabs %}
+
+### Examples
+
+<!-- vale Google.Headings = NO -->
+#### `demo-app` communicates only with `redis` on port 6379
+<!-- vale Google.Headings = YES -->
+
+{% tabs reachable-backends useUrlFragment=false %}
+{% tab reachable-backends Kubernetes %}
+```yaml
+apiVersion: apps/v1
+kind: Pod
+metadata:
+  name: demo-app
+  namespace: kuma-demo
+  annotations:
+    kuma.io/reachable-backends: |
+      refs:
+      - kind: MeshService
+        name: redis
+        namespace: kuma-demo
+        port: 6379
+...
+```
+{% endtab %}
+{% tab reachable-backends Universal %}
+```yaml
+type: Dataplane
+mesh: default
+name: {% raw %}{{ name }}{% endraw %}
+networking:
+  address: {% raw %}{{ address }}{% endraw %}
+  inbound:
+  - port: {% raw %}{{ port }}{% endraw %}
+    tags:
+      kuma.io/service: demo-app
+  transparentProxying:
+    redirectPortInbound: {{ tproxy.defaults.redirect.inbound.port }}
+    redirectPortOutbound: {{ tproxy.defaults.redirect.outbound.port }}
+    reachableBackends:
+      refs:
+      - kind: MeshService
+        name: redis
+        port: 6379
+```
+{% endtab %}
+{% endtabs %}
+
+<!-- vale Google.Headings = NO -->
+#### `demo-app` doesn’t need to communicate with any service
+<!-- vale Google.Headings = YES -->
+
+{% tabs reachable-backends-no-services useUrlFragment=false %}
+{% tab reachable-backends-no-services Kubernetes %}
+```yaml
+apiVersion: apps/v1
+kind: Pod
+metadata:
+  name: demo-app
+  namespace: kuma-demo
+  annotations:
+    kuma.io/reachable-backends: ""
+...
+```
+{% endtab %}
+{% tab reachable-backends-no-services Universal %}
+```yaml
+type: Dataplane
+mesh: default
+name: {% raw %}{{ name }}{% endraw %}
+networking:
+  address: {% raw %}{{ address }}{% endraw %}
+  inbound:
+    - port: {% raw %}{{ port }}{% endraw %}
+      tags:
+        kuma.io/service: demo-app
+  transparentProxying:
+    redirectPortInbound: {{ tproxy.defaults.redirect.inbound.port }}
+    redirectPortOutbound: {{ tproxy.defaults.redirect.outbound.port }}
+    reachableBackends: {}
+```
+{% endtab %}
+{% endtabs %}
+
+<!-- vale Google.Headings = NO -->
+#### `demo-app` communicates with all MeshServices in the `kuma-demo` namespace
+<!-- vale Google.Headings = YES -->
+
+{% tabs reachable-backends-in-namespace useUrlFragment=false %}
+{% tab reachable-backends-in-namespace Kubernetes %}
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: demo-app
+  namespace: kuma-demo
+  annotations:
+    kuma.io/reachable-backends: |
+      refs:
+      - kind: MeshService
+        labels:
+          k8s.kuma.io/namespace: kuma-demo
```

Comment:
There can be a universal equivalent to this right?

A universal workload that only talks to stuff in a kuberenetes namespace (in another zone)